File: translations/codenet/deepseek_commented/Go2J/gpt-4o/Code/s695527706.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in); // Creating a Scanner object for input
        int n = scanner.nextInt(); // Reading the integer input
        String s = scanner.next(); // Reading the string input
        int count = countOccurrences(s, "ABC"); // Counting occurrences of "ABC" in s
        System.out.println(count); // Printing the result
        scanner.close(); // Closing the scanner
    }

    // Method to count occurrences of a substring in a string
    private static int countOccurrences(String str, String subStr) {
        int count = 0;
        int index = 0;
        while ((index = str.indexOf(subStr, index)) != -1) {
            count++;
            index += subStr.length(); // Move past the last found occurrence
        }
        return count;
    }
}

// <END-OF-CODE>

File: translations/codenet/deepseek_commented/Go2C/gpt-4o/Code/s695527706.c
#include <stdio.h>
#include <string.h>

int countOccurrences(const char *str, const char *sub) {
    int count = 0;
    const char *tmp = str;
    while ((tmp = strstr(tmp, sub)) != NULL) {
        count++;
        tmp++; // Move past the last found occurrence
    }
    return count;
}

int main() {
    int n; // Declaring an integer variable to store the number of inputs
    char s[100]; // Declaring a string variable to store the input string (assuming max length of 99)
    
    scanf("%d %s", &n, s); // Reading input from the user and storing it in n and s
    printf("%d\n", countOccurrences(s, "ABC")); // Counting the occurrences of "ABC" in s and printing the result

    return 0;
}

// <END-OF-CODE>
